modules/virt-installing-qemu-guest-agent-on-linux-vm.adoc

```diff
@@ -7,14 +7,13 @@
 [id="virt-installing-qemu-guest-agent-on-linux-vm_{context}"]
 = Installing the QEMU guest agent on a Linux VM
 
-The `qemu-guest-agent` is widely available and available by default in {op-system-base-full} virtual machines (VMs). Install the agent and start the service.
+The `qemu-guest-agent` is available by default in {op-system-base-full} virtual machines (VMs)
 
-[NOTE]
-====
-To create snapshots of an online (Running state) VM with the highest integrity, install the QEMU guest agent.
+To create snapshots of a VM in the `Running` state with the highest integrity, install the QEMU guest agent.
 
-The QEMU guest agent takes a consistent snapshot by attempting to quiesce the VM file system as much as possible, depending on the system workload. This ensures that in-flight I/O is written to the disk before the snapshot is taken. If the guest agent is not present, quiescing is not possible and a best-effort snapshot is taken. The conditions under which the snapshot was taken are reflected in the snapshot indications that are displayed in the web console or CLI.
-====
+The QEMU guest agent takes a consistent snapshot by attempting to quiesce the VM file system. This ensures that in-flight I/O is written to the disk before the snapshot is taken. If the guest agent is not present, quiescing is not possible and a best-effort snapshot is taken. 
+
+The conditions under which a snapshot is taken are reflected in the snapshot indications that are displayed in the web console or CLI. If these conditions do not meet your requirements, try creating the snapshot again, or use an offline snapshot
```

Comment:
```suggestion
The conditions under which a snapshot is taken are reflected in the snapshot indications that are displayed in the web console or command-line interface (CLI). If these conditions do not meet your requirements, try creating the snapshot again, or use an offline snapshot
```

- First time occurrence of CLI, alternatively you might want to just use "command-line interface" without the acronym if CLI is being mentioned only once.